content/artifacts/artifactory/plugin.js

```diff
@@ -245,7 +245,7 @@ function Unfound({ selected }) {
   let [lastLocationId, setLastLocationId] = useState(null);
 
   let planets = myPlanetsWithFindable()
-    .filter(planet => !planet.hasTriedFindingArtifact && !planet.unconfirmedProspectPlanet && planet.prospectedBlockNumber === undefined ) ;
+    .filter(planet => !planet.hasTriedFindingArtifact && (planet.prospectedBlockNumber === undefined || planet.hasOwnProperty('unconfirmedFindArtifact')));
```

Comment:
Are you sure the planet has property "unconfirmedFindArtifact" when it finish prospect(means prospectedBlockNumber is not undefined) BUT not start find?
If you want to hide the prospect expired planets, I suggest change this line to:
```js
.filter(planet => !planet.hasTriedFindingArtifact && (planet.prospectedBlockNumber === undefined || !prospectExpired(currentBlockNumber, planet.prospectedBlockNumber)))
```

Reply:
Yes, i'm sure the planet has the property "unconfirmedFindArtifact" after the prospect. 
I think your way is better to hide the prospect expired planets. I've made the change you suggested. Thanks.